src/qmozview_p.cpp

```diff
@@ -1518,6 +1523,17 @@ bool QMozViewPrivate::handleAsyncMessage(const QString &message, const QVariant
         QInputMethod *inputContext = qGuiApp->inputMethod();
         inputContext->update(Qt::ImSurroundingText | Qt::ImCursorPosition | Qt::ImAnchorPosition);
         return true;
+    } else if (message == QLatin1String(INPUTMETHOD_SET_INPUT_ATTRIBUTES)) {
+        QVariantMap map = data.toMap();
+        mInputMethodAttributes = 0;
+        if (map.value(QLatin1String("autocomplete")) == "off") mInputMethodAttributes |= Qt::ImhNoPredictiveText;
+        if (map.value(QLatin1String("autocapitalize")) == "off") mInputMethodAttributes |= Qt::ImhNoAutoUppercase;
```

Comment:
This should "off" or "none" [1]. Further, attribute states that all letters defaults to lowercase. Hence, I think this should also have Qt::ImhPreferLowercase.

In addiotn, this could cover value "characters" with Qt::ImhPreferUppercase.

[1]
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize